src/applications/find-va-forms/tests/containers/SearchResults.unit.spec.jsx

```diff
@@ -19,6 +19,14 @@ describe('Find VA Forms <SearchResults>', () => {
     tree.unmount();
   });
 
+  it('renders an error alert box', () => {
+    const tree = shallow(<SearchResults error="test" />);
+
+    expect(tree.html()).to.include('test');
```

Comment:
Since this test is for the `<AlertBox>` appearing maybe the test should also look for "Something went wrong" to make it a bit more obvious.